binr/rafind2/rafind2.c

```diff
@@ -90,6 +91,7 @@ static int show_help(char *argv0, int line) {
 	" -f [from]  start searching from address 'from'\n"
 	" -h         show this help\n"
 	" -i         identify filetype (r2 -nqcpm file)\n"
+	" -d         describe results with the filename (useful for recursive search)\n"
```

Comment:
what about using -q to 'quiet' and make this 'describe' thing enabled by default?

Reply:
Sure! Enabling this by default makes sense, I am not sure about the quiet option though, quiet seems like it should do more than just suppressing printing the filename (I could do that to but what is to be suppressed need to be defined).
What about just using options _similar_ to what `grep` offers?
```
       -H, --with-filename
              Print the file name for each match.  This is the default when there is more than one file to search.

       -h, --no-filename
              Suppress the prefixing of file names on output.  This is the default when there is only one file (or only standard input) to search.
```